docs/repository-management.md

```diff
@@ -0,0 +1,134 @@
+# Repository Management
+
+The goal is for this to be a living document explaining how we collaboratively manage the Gutenberg repository. If you’d like to suggest a change, please open an issue for discussion or submit a pull request to the document.
+
+This document covers:
+
+- [Issues](#issues)
+  - [Labels](#labels)
+  - [Milestones](#milestones)
+  - [Triaging Issues](#triaging-issues)
+- [Pull Requests](#pull-requests)
+  - [Code Review](#code-review)
+  - [Design Review](#design-review)
+  - [Merging Pull Requests](#merging-pull-requests)
+  - [Closing Pull Requests](#closing-pull-requests)
+
+## Issues
+
+A healthy issue backlog is one where issues are relevant and actionable. *Relevant* in the sense that they relate to the project’s current priorities. *Actionable* in the sense that it’s clear what action(s) need to be taken to resolve the issue.
```

Comment:
Another descriptor for healthy issue backlog:

- Specific: No broad mega-issues, because they're difficult to close. Ideally an issue can be closed in a single pull request.

Reply:
Maybe under "Actionable", making clear the steps to reproduce a particular issue. It's difficult and time-consuming to debug an issue when it's specific to a set of steps.